libs/dal/src/repositories/message/message.repository.ts

```diff
@@ -451,17 +531,13 @@ export class MessageRepository extends BaseRepository<MessageDBModel, MessageEnt
    * unread -> { seen: true, read: false, archived: false }
    * unarchived -> { seen: true, read: true, archived: false }
    */
-  async updateMessagesStatus({
-    environmentId,
-    subscriberId,
-    ids,
+  private async updateMessagesStatus({
+    query,
     seen,
     read,
     archived,
   }: {
-    environmentId: string;
-    subscriberId: string;
-    ids: string[];
+    query: MessageQuery & EnforceEnvId;
```

Comment:
`MessageQuery & EnforceEnvId` these types are set on the repository, hence the query is using it